services/movie_session.py

```diff
@@ -0,0 +1,44 @@
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: int,

```

Comment:
The parameter `movie_show_time` should be of type `datetime` instead of `int`. Update the function signature to `movie_show_time: datetime`.